### DISCUSSION TITLE
Rogue cluster in 10X single cell sequencing data!

### DISCUSSION THREAD

**Discussion by fmclean90** [satijalab/seurat#6742] [Q&A]
Hello,

I am carrying out 10X single-cell RNA seq analysis. I have clustered my cells and was happy with the way it looked and had begun to identify my cells when I came across a rogue cluster (cluster 8). 

When I look in the top10 expressed genes to help identify the cluster **it stopped after identifying a top 4 for that cluster**. I wondered if these were dead/dying endothelial cells, but I filter out mitochondrial content at over 10% and I have run cell death and cell senescent markers and none of them are expressed in this group. Weirdly, **it does have expression for 2 of my key identifying markers**, even though neither of these come up in the top10 expressed clusters. I noticed it has more lower features than almost all the other clusters (I currently filter out 200 features). 

I spoke to some bioinformatics colleagues and they suggested that it is low quality cells and to try running the pipelines with more stringent filtering. I've tried upping the minimum number of features to 500. However, is it right to filter out the low-quality cells and jeopardising losing rarer cell types? Also, is there advice on mitochondrial filtering. I have it set at 10% but don't want to risk losing any cells which have high respiration.

Has anyone had a similar problem and how is it best to manage it? Any ideas why the cluster only has a top 4 expressed genes but still shows expression for particular cell markers?

Any help would be greatly appreciated!

![Clusters](https://user-images.githubusercontent.com/119618724/205121723-2f76351c-89a5-4c00-aefc-511f37edc56e.png)
![Genes](https://user-images.githubusercontent.com/119618724/205121731-c966df17-e915-4da4-8ad8-c91a462f1523.png)

**Comment by jietian-327** [comment]
I have the same question. I think it is some low quality double cells. I choose to delete them manually. Please let me know if I'm wrong.
